.github/workflows/dependency-graph.yml

```diff
@@ -2,13 +2,16 @@ name: Update Dependency Graph
 on:
   push:
     branches:
-      - main # default branch of the project
+      - ts/update-sbt-dependency-workflow # for testing
+      # - main # default branch of the project
 jobs:
   dependency-graph:
     name: Update Dependency Graph
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
       - uses: scalacenter/sbt-dependency-submission@v2
+        with:
+          configs-ignore: test test-internal compile compile-internal scala-tool scala-doc-tool
```

Comment:
Do we want to ignore `compile`/`compile-internal` config? It seems to get replaced by `runtime` config when ignored. @adamnfish

Reply:
Actually, after discussion with @NovemberTang, we have decided to leave the workflow at default, and exclude any dev dependencies at the repocop stage, so that we can change the level of monitoring centrally (i.e. if we did decide to alert on dev dependencies in the future).